sky/setup_files/setup.py

```diff
@@ -168,7 +168,11 @@ def parse_readme(readme: str) -> str:
     install_requires=dependencies['install_requires'],
     extras_require=dependencies['extras_require'],
     entry_points={
-        'console_scripts': ['sky = sky.cli:cli'],
+        'console_scripts': [
+            # Install skyserver to venv to pin the module used, refer to #4801
+            'sky = sky.cli:cli',
+            'skyserver = sky.server.server:main'
```

Comment:
Hmm, I am not very sure about exposing this. For ray's setup, the CLI they expose is the user-facing library instead of an internal server.

One potential way to fix this is to just get find the absolute path and start the server directly with the absolute path.
```python
server_path = os.path.join(sky.__root_dir__, 'server', 'server')
```

Reply:
Good idea! Looking into this